docs/developer/order-expiration.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows finalizing checkout without payment. This can be achieved in two ways:
+
+- by setting `orderSettings.allowUnpaidOrders` for a given `Channel` to `true` and calling `checkoutComplete` without any payment/transaction object
+- by using the `orderCreateFromCheckout` mutation with first order then payment flow.
```

Comment:
I would add `protected` here - to clarify that creating an order via this mutation requires permission. 

`with first order then payment flow.` -> I would change it to something as you have in previous step: `without any payment/transaction object` (would be also good to use names from api-reference, (and links them with api-reference section): `Payment`/`TransactionItem` object)